setup.py

```diff
@@ -267,7 +267,7 @@ def initialize_options(self):
         self._clean_trees = []
         self._clean_exclude = ['np_datetime.c',
                                'np_datetime_strings.c',
-                               'period.c',
+                               'src/period.c',
```

Comment:
Should I remove this from #9415 and use full relative paths for ignore? e.g. `pandas/src/period.c` and `pandas/src/datetime/np_datetime.c` in `self._clean_exclude`.